### ISSUE TITLE
PATH ignored in MacOS for SQLite3 connections

### ISSUE THREAD

**Issue by ts1g** [TablePlus/TablePlus#2767]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): SQLite3 v 3.39.2

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 436

3. The steps to reproduce this issue: MacOS currently uses SQLite3 v 3.37, while v 3.39.x now (finally!) offers FULL JOIN. Since /usr/bin where macOS SQLite v 3.37 resides is protected, it is necessary to install newer versions of SQLite3 in /usr/local/bin and then set the PATH in BASH or ZSH profile to a) point to the newer version of SQLite and b ) set priority of newer version at /usr/local/bin as higher than the /usr/bin version installed at the base macOS.  

Updating PATH to push newer SQLite version to top of priority list works for Terminal and other programs (confirmed by using which -a sqlite in Terminal and sqlite_version() in other apps) but TablePlus continues to return the older version of SQLite when sqlite_version() used. Is there to force TablePlus to honor PATH priority?

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
Hi @ts1g TablePlus is a stand-alone application and it does not need any dependencies. SQLite is bundled with the app, so it has nothing to do with the SQLite in /usr/local/bin or anywhere in the machine.
We're working to update the SQLite to the latest release. (It's a bit late because TablePlus also supports the SQLite cipher).

**Comment by ts1g** [comment]
Hi @huyphams not saying it needs dependencies; saying that there's no way to force TablePlus to honor the PATH priority either automatically (ideal) or manually (when I set up a local connection to SQLite, allow me to specify which one of multiple versions of SQLite's engine on my Mac that I want the db open with...

**Comment by huyphams** [comment]
If we allow the user to sideload any dynamic library, it would make the app less safe and easy for attackers to hijack into the app.

**Comment by ts1g** [comment]
Fair point; but it's not unlike connections to any other non-embedded local database. Regardless, look forward to being able to do FULL JOIN if you're going to include SQLite 3.39.x in TablePlus as it's been out for a bit and SQLite is one of the few DBs that still requires massive workarounds for joins...

**Comment by king724** [comment]
Build 472 Release date: 25 December 2022 has Sqlite 3.39.2! 
https://tableplus.com/blog/2017/02/changelogs.html

Thank you!
